coalib/bearlib/languages/definitions/DOT.py

```diff
@@ -0,0 +1,19 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class DOT:
+    aliases = 'DOT',
+    extensions = '.gv',
+    comment_delimiters = '//',
+    multiline_comment_delimiters = {'/*': '*/'}
+    string_delimiters = {"'": "'"}
+    indent_types = {'{': '}'}
+    encapsulators = {'(': ')', '[': ']'}
```

Comment:
Also add < and > delimiter here. Its given [here](https://www.graphviz.org/doc/info/lang.html)